app/src/main/java/com/tapglue/tapgluetests/TestActivity.java

```diff
@@ -600,8 +603,26 @@ public void onRequestFinished(@Nullable TGFeed output, boolean changeDoneOnline)
                         }
 
                         testController.log("#3.5 finished correctly");
-                        // TODO: after fix of delete request change this to TEST_3_6
-                        doTest(TEST_3_7_1, randomUserName, randomUserName2, runnable);
+                        TGQuery query = new TGQuery();
+                        query.setObject(new TGQueryTGObject().setId(new TGQueryObject<String>().setEq("hello")));
```

Comment:
Im a little confused, whats the difference between TGQueryObject and TGQueryTGObject ??